pkg/controller/controller.go

```diff
@@ -480,10 +480,13 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) {
 			}
 		}
 
+		ns, _ := c.kubeClient.Core().Namespaces().Get(instance.Namespace)
```

Comment:
@pmorie We get an empty string in the case of erroring, which I think is fine to populate the OrgID and SpaceID with. IMO generating a new GUID might result in this field being useless since the users will not able to confidently trust the NS actually exists.